test/e2e/flowexporter_test.go

```diff
@@ -235,8 +292,71 @@ func checkRecordsWithPodIPs(t *testing.T, data *TestData, podAIP string, podBIP
 			}
 		}
 	}
-	assert.Equal(t, templateRecords, clusterInfo.numNodes, "Each agent should send out template record")
+	assert.GreaterOrEqual(t, templateRecords, clusterInfo.numNodes, "Each agent should send out template record")
```

Comment:
Just curious on why do we need to move to GreaterOrEqual? Will there be more template records?

Reply:
Sometimes there will be more than 1 template record from the same node.

Reply:
Is the reason because of antrea agent restarts or ipfix-collector pod restarts? Or something else?

Reply:
After checking, I found that after each test, it will tear down the resources under namespace `antrea-test`, but will not change antrea agent config back to before. So antrea agent will keep trying to send records to collector ip. Most of the time the collector will deploy with the same ip (curious about why). Between ipfix-collector is ready and  antrea agent gets restarted, the collector will probably receive template records.

Reply:
Thanks for looking into this. May be we should not specify masternode every time we deploy the ipfix-collector pod; give nodename as an empty string. Also, will `since` option help in getting the exact logs with [kubectl logs](https://jamesdefabia.github.io/docs/user-guide/kubectl/kubectl_logs/)?

Reply:
After setting nodename as empty string and rearranged the test cases as Antonin suggested. Since it will only setup once, it works with `Equal` now. Thanks!